docs/refactoring/course-read-model/scripts/scale_replay.py

```
@@ -0,0 +1,181 @@
+
+def jitter_pos(lat, lng):
+    return (lat + random.gauss(0, 0.25 / 111.0),
+            lng + random.gauss(0, 0.25 / (111.0 * math.cos(math.radians(lat)))))
+
+
+def amplify(base_events, base_runs, scale):
+    evs, rns = [], []
+    for c in range(scale):
+        for e in base_events:
+            la, lo = (e["lat"], e["lng"]) if c == 0 else jitter_pos(e["lat"], e["lng"])
+            t = e["ts"] if c == 0 else e["ts"] + random.uniform(-1800, 1800)
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)